packages/codebytes/src/editor.tsx

```diff
@@ -36,18 +37,16 @@ const DOCKER_SIGTERM = 143;
 
 type EditorProps = {
   hideCopyButton: boolean;
-  language: languageOption;
+  language: LanguageOption;
   text: string;
   onChange: (text: string) => void;
-  onCopy?: (text: string, language: string) => void;
```

Comment:
We still will want to keep this prop, since, the copy behavior for forums is slightly different from the generic copy behavior :)

Reply:
(We'll ensure the CodeByteEditor component accepts onCopy? as a prop and pass it into Editor)